packages/web/styles/globals.css

```diff
@@ -314,3 +314,18 @@
   z-index: 9000000;
   box-shadow: none !important;
 }
+
+.breakspaces {
+  white-space-collapse: break-spaces;
+}
+
+.aftershadow::after {
```

Comment:
There is the same css prop repeated, but you can alone move this code to tailwind classes, you don't need a custom one.

You can define after element and also gradient, I don't like the current solution because colors are hardcoded here.

Reply:
I moved it to a css rule in order to avoid cluttering the classNames in the component

Reply:
Ye it makes sense, but you can use "@apply" for that: https://tailwindcss.com/docs/reusing-styles#extracting-classes-with-apply